src/Conf.ml

```diff
@@ -985,6 +986,17 @@ module Formatting = struct
     C.flag ~default ~names ~doc ~section
       (fun conf wrap_fun_args -> {conf with wrap_fun_args})
       (fun conf -> conf.wrap_fun_args)
+
+  let end_of_line_in =
+    let default = false in
+    let doc =
+      "Place $(i,in)s (from $(i,let)-$(i,in)-s) at the end of the last \
+       line of the expression it delimits instead of on a line of its own."
+    in
+    let names = ["end-of-line-in"] in
```

Comment:
Considering symmetry with the other option names, I think it would be better to use e.g. `no-break-before-in`. What do others think?

Reply:
I prefer when options are written as positive in the source code and configuration files (`break-before-in=false` is ok `no-break-before-in=true` is not).

I don't particularly mind about the default (`true` or `false`, leading to `--no-X` or `--X`) for command-line arguments.

Reply:
The current behavior should be the default (for at least one release). Avoiding `no-` prefixes in the source would be good. It would be good to check that the docstring is phrased so that it makes sense from the perspective of changing from the default when reading the output of --help.

Reply:
I renamed the option and rephrased the docstring. The docstring is written for the positive option because of the option name (`break-before-in`). However, because it defaults to `true` the option is presented in the help/man as `--no-break-before-in`.